Lib/defconQt/fontInfo.py

```diff
@@ -180,22 +180,24 @@ class GeneralTab(TabWidget):
 
     def __init__(self, font, parent=None):
         super().__init__(parent, name="General")
+        self.name = self.tr("General")
```

Comment:
This duplication needs to go. Can you just remove the name arg from \__init__ call?